src/main/java/core/stack/MateStack.java

```diff
@@ -45,4 +47,12 @@ private T getElement() {
         isEmptyStack();
         return stack[getIndexOfLastElement()];
     }
+
+    private boolean resize() {
```

Comment:
do you use this return type?

Reply:
Bogdan said that better not to use void methods because it is harder to test

Reply:
will you test private methods?